yrs/src/update.rs

```diff
@@ -609,7 +609,7 @@ impl Update {
                                 } else {
                                     // if (currWrite.struct.id.clock + currWrite.struct.length >= curr.id.clock) {
                                     let diff =
-                                        curr_write_last as i32 - curr_block.id().clock as i32;
+                                        curr_write_last as i32 - curr_block.id().clock as i32 - 1;
```

Comment:
How this variable is computed in Yjs:

```js
const diff = currWrite.struct.id.clock + currWrite.struct.length - curr.id.clock
```

Thing is that the difference between `struct.id.clock + struct.length` and `struct.last_id.clock` is always `1`.